test/unit/math/prim/fun/interp1_test.dSYM/Contents/Info.plist

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
Delete this file and also `test/unit/math/prim/fun/interp1_test.dSYM/Contents/Resources/DWARF/interp1_test`.

It's worth double checking what you're committing, since once it gets committed, it stays in the repo history even if we revert it.

Reply:
ok, I will double check